Libraries/oneMKL/fourier_correlation/makefile

```diff
@@ -0,0 +1,24 @@
+# Makefile for NMAKE
+
+default: run_all
+
+all: run_all
+
+run_all: fcorr_1d_buff.exe fcorr_1d_usm.exe
+	.\fcorr_1d_buff 1024
+	.\fcorr_1d_usm 1024
+
+DPCPP_OPTS=-DMKL_ILP64  -I"%MKLROOT%\include" mkl_sycl_dll.lib mkl_intel_ilp64_dll.lib mkl_tbb_thread_dll.lib mkl_core_dll.lib tbb.lib sycl.lib OpenCL.lib

```

Comment:
Hi @hagabb, for windows mkl_sycl_dll.lib supports only blas and lapack domains, so you need to use static mkl_sycl.lib